table/tables/tables.go

```diff
@@ -147,6 +148,13 @@ func TableFromMeta(allocs autoid.Allocators, tblInfo *model.TableInfo) (table.Ta
 				return nil, err
 			}
 			col.GeneratedExpr = expr
+			newCol := col
+			col.GetGeneratedExpr = func() ast.ExprNode {
+				// The error is checked before, so we can ignore them here.
+				expr, _ := generatedexpr.ParseExpression(newCol.GeneratedExprString)
+				expr, _ = generatedexpr.SimpleResolveName(expr, tblInfo)
+				return expr
```

Comment:
Is it possible that `tblInfo` was modified before `GetGeneratedExplr` is called?

Reply:
I'm not sure, but the newest schema is correct anyway.

Reply:
I mean the error may not be checked in this case.